_projects/2024/100546612/100546612.Rmd

```diff
@@ -0,0 +1,1007 @@
+                  big.mark = ".",prefix = "$", decimal.mark = "none"),
+                limits = c(800, 57000), 
+                
+                # expanding panels: width
+                expand = expansion(add = c(0.17, 0.05))
+                ) + 
+  
+  scale_y_continuous(breaks = c(0, 25, 50, 75, 100),
+                     minor_breaks = NULL,
+                     labels = scales::percent_format(scale = 1), 
+                     limits = c(0, 100),
+                     
+                     # expanding panels: height
+                     expand = expansion(mult = c(0.08, 0.04))
+                     ) +
+
+  scale_color_manual(values = colour_scheme) + 
+  geom_path(data = data_filtered_2,
+            aes(x = GDP, y = poverty, 
+                group = region2, 
+                color = region2), 
+            alpha = 0.2, 
+            linewidth = 0.2) + 
+  geom_path(linewidth = 0.35) + 
+  facet_wrap(~ region, scales = "free") +
+  theme_minimal() + 
+  theme(
+    legend.position = "none", 
+    strip.text = element_blank(),
+    axis.title = element_blank(), 
+    
+    # axes guides theme 
+    axis.text.x = element_text(
+      family = "lato", colour = "#939292", size = 26.5, 
+      hjust = c(0.4, 0.4, 0.4, 0.4, 0.4, 0.6), margin = margin(t = -10)), 
+    axis.text.y = element_text(
+      family = "lato", colour = "#676767", size = 26.5, 
+      vjust = 0.5, margin = margin(r = -16)),
+
+    # panel theme 
+    panel.background = element_rect(fill = "#faf9f9"), 
+    panel.border = element_rect(fill = NA, color = "#efefef"), 
+    
+    # adjust between-panel spacing
+    panel.spacing.x = unit(0.45, "lines"), 
+    panel.spacing.y = unit(0.6, "lines"), 
+    
+    panel.grid.major = element_blank()
+    ) 
+
+plot_theme
+```
+
+### Titles
+
+Followingly, we include the **title, subtitle, data sources (caption) and the license information**, taking into consideration their different fonts, sizes and colours, i.e. their individual themes. Take into consideration that the "-" appear in R as the length -, even when they are actually of length – or —.
+
+```{r titles, layout="l-body-outset",  fig.width = 8, fig.height = 5.8, fig.dpi = 300}
+# library(ggtext)
+
+# text labels
+graph_title <- "By world region: Share living in extreme poverty vs. GDP per capita (1820-2017)"
+
+graph_subtitle <- "Poverty is measured by the 'cost of basic needs'-approach. It represents the share which was unable to obtain a bundle of very basic goods and services.\nPrice changes over time (inflation) and price differences between countries are taken into account in both measures."
+
+graph_caption <- "Data sources:  Michail Moatsos for data on extreme poverty and Jutta Bolt and Jan Luiten van Zanden for GDP data – both published in How was life? Volume II, OECD 2021.<br><span style = 'font-family:lato; color:#403c94;'>OurWorldinData.org</span> – Research and data to make progress against the world's largest problems."
```

Comment:
The trick to avoid long lines of text going off the page is to do something like

```r
paste(c(
  "my",
  "long",
  "line",
  "of text"
), collapse=" ")
```